Sources/Core/Components/TimerFactory.swift

```diff
@@ -14,13 +14,42 @@ protocol TimerProtocol {
 
 extension Timer: TimerProtocol { }
 
+final class TimerAdapter: TimerProtocol {
```

Comment:
Maybe we could put that class as an innerclass of TimerFactory, in order to change its visibility to private.
What do you think?

Reply:
yep, good idea